src/components/Main/Navigation/ChannelList/ChannelElementHash.vue

```diff
@@ -105,26 +106,123 @@ export default defineComponent({
     &[data-is-opened] {
       color: $theme-background-secondary;
       background: $theme-ui-primary;
+      &:hover {
+        &::before {
+          content: '';
+          border: {
+            width: 2px;
+            style: solid;
+            color: transparent;
+          }
+          border-radius: 4px;
+          width: 140%;
+          height: 140%;
+          display: block;
+          position: absolute;
+          top: -4px;
+          left: -4px;
+          background: $theme-ui-secondary;
+          opacity: 0.5;
+        }
+      }
```

Comment:
```suggestion
    &:hover {
      &::before {
          content: '';
          border: {
            width: 2px;
            style: solid;
            color: transparent;
          }
          border-radius: 4px;
          width: 140%;
          height: 140%;
          display: block;
          position: absolute;
          top: -4px;
          left: -4px;
      }
    }
    &[data-is-opened] {
      color: $theme-background-secondary;
      background: $theme-ui-primary;
      &:hover {
        &::before {
          background: $theme-ui-secondary;
          opacity: 0.5;
        }
      }
```
こんな風に共通な箇所は上の階層で書くことで記述量を減らせます
こうすることでほかの箇所も`background`と`opacity`だけ記述すればよいようになると思います